packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/date/CommonDateRange.tsx

```diff
@@ -0,0 +1,149 @@
+import {Flex, Stack} from '@sanity/ui'
+import {addDays} from 'date-fns'
+import React, {useCallback, useMemo} from 'react'
+import {useSearchState} from '../../../../../contexts/search/useSearchState'
+import type {OperatorDateRangeValue} from '../../../../../definitions/operators/dateOperators'
+import type {OperatorInputComponentProps} from '../../../../../definitions/operators/operatorTypes'
+import {DateIncludeTimeFooter} from './dateIncludeTimeFooter/DateIncludeTimeFooter'
+import {DatePicker} from './datePicker/DatePicker'
+import {ParsedDateTextInput} from './ParsedDateTextInput'
+import {getDateISOString} from './utils/getDateISOString'
+
+export function CommonDateRangeInput({
+  isDateTime,
+  onChange,
+  value,
+}: OperatorInputComponentProps<OperatorDateRangeValue> & {
+  isDateTime: boolean
+}) {
+  const {
+    state: {fullscreen},
+  } = useSearchState()
+
+  const placeholderStartDate = useMemo(() => addDays(new Date(), -7), [])
```

Comment:
I guess we subtract 7 days from today to get a placeholder date that is in the past/one week back in time. Would be nice with a small comment explaining why we do this :)